docs/modules/ROOT/pages/guide-keep3r.adoc

```diff
@@ -0,0 +1,138 @@
+const workableAbi = [{"anonymous":false,"inputs":[{"indexed":false,"internalType":"address","name":"worker","type":"address"}],"name":"Worked","type":"event"},{"inputs":[],"name":"requestWork","outputs":[],"stateMutability":"nonpayable","type":"function"},{"inputs":[],"name":"work","outputs":[],"stateMutability":"nonpayable","type":"function"},{"inputs":[],"name":"workable","outputs":[{"internalType":"bool","name":"","type":"bool"}],"stateMutability":"view","type":"function"}];
+
+const { ethers } = require("ethers");
+const { DefenderRelaySigner } = require('defender-relay-client/lib/ethers');
+
+async function workIfNeeded(contract) {
+  if (await contract.workable()) {
+    const tx = await contract.work();
+    console.log(`Worked ${contract.address}:`, tx.hash);
+  } else {
+    console.log(`No work needed for ${contract.address}`);
+  }
+}
+
+function getProvider() {
+  return ethers.getDefaultProvider('homestead', {
+    etherscan: etherscanToken === 'NONE' ? null : etherscanToken,
+    infura: infuraToken === 'NONE' ? null : infuraToken,
+    alchemy: alchemyToken === 'NONE' ? null : alchemyToken,
+  })
+}
+
+// Entrypoint for the Autotask
+exports.handler = async function(credentials) {
+  const provider = getProvider();
+  const signer = new DefenderRelaySigner(credentials, provider, { speed: 'fastest', from });
+  const registry = new ethers.Contract(registryAddr, registryAbi, signer);
+  const contract = new ethers.Contract(workableAddr, workableAbi, signer);
+
+  // Work if is a registered keeper
+  if (await registry.isKeeper(from)) {
+    return await workIfNeeded(contract);
+  }
+
+  // Otherwise run through bonding and activation process
+  const bonding = await registry.bondings(from, collateralAddr).then(b => b.toNumber());
+  if (bonding === 0) {
+    const tx = await registry.bond(collateralAddr, collateralAmount);
+    console.log(`Bonded relayer: ${tx.hash}`);
+  } else if (bonding < Date.now() / 1000) {
+    const tx = await registry.activate(collateralAddr);
+    console.log(`Activated relayer: ${tx.hash}`);
+    await workIfNeeded(contract);
+  } else {
+    console.log(`Waiting ${bonding - parseInt(Date.now() / 1000)} seconds until activation is available`);
+  }
+}
+
+// To run locally (this code will not be executed in Autotasks)
+if (require.main === module) {
+  const { API_KEY: apiKey, API_SECRET: apiSecret } = process.env;
+  exports.handler({ apiKey, apiSecret })
+    .then(() => process.exit(0))
+    .catch(error => { console.error(error); process.exit(1); });
+}
+----
+
+## Wait for it!
+
+The script above will take care of *bonding* your Relay to the keep3r network using zero KPR tokens as collateral. After bonding, there is a three-day delay enforced by the network before you can start using your Keeper. Once that time has passed, the script above will take care of *activating* your Relay as a Keeper.
```

Comment:
Is this required per Keeper task or is it a single registration with the Keeper Network?

If it is a single registration, could we split the registration into its own Autotask?  
If it is a registration per Keeper task, then we should add a note of this.  We could also put comments around the registration code so that this can be used again for other tasks.

Reply:
We are working to handle registration directly on the Relayer, so this is does not pollute the Autotask. For now, I preferred to include everything in a single autotask to simplify setup - user needs to copy-paste only one code snippet.

Reply:
That said, agree on the comments.